Cargo.toml

```diff
@@ -216,10 +216,15 @@ opt-level = 'z'
 opt-level = 3
 lto = "fat"
 codegen-units = 1
-strip = "symbols" # Set to `false` for debug information
-debug = false # Set to `true` for debug information
```

Comment:
Can I keep the fields? It's easier to reason about the profile when everything is explicitly listed so we don't need to look up the docs for what the defaults are.